GVFS/GVFS.Common/ProductUpgrader.Shared.cs

```diff
@@ -11,7 +11,7 @@ public partial class ProductUpgrader
         public const string DownloadDirectory = "Downloads";
 
         private const string RootDirectory = UpgradeDirectoryName;
-        private const string GVFSInstallerFileNamePrefix = "SetupGVFS";
+        private const string GVFSInstallerFileNamePrefix = "VFSGit";

```

Comment:
Are you planning on updating the actual installer in this PR too? If not, do we need to support both format?

Reply:
>>Are you planning on updating the actual installer in this PR too?

Yes - renamed the actual installer.